### DISCUSSION TITLE
flashing gsi kills touch

### DISCUSSION THREAD

**Discussion by vkrathi042** [phhusson/treble_experimentations#1941] [Q&A]
sir i tried to flash gsi on my realme 7 pro , like for example recent one is dot os but it kills my touch nd i has to use mouse to flash stock to get touch working    any solution to this please resppond

**Comment by phhusson** [comment]
That's on my aosp v310?

20 juil. 2021 16:36:03 vkrathi042 ***@***.***>:

**Reply by vkrathi042** [reply]
![Uploading Screenshot_20210720-221921800.jpg…]()

**Reply by vkrathi042** [reply]
sorry sir ...please see this  screenshot... i am still learning so i don't understand what you are saying sorry for inconvenience

**Reply by vkrathi042** [reply]
project treble using vndk version 30.0

**Comment by phhusson** [comment]
Which gsi did you flash?

20 juil. 2021 18:59:22 vkrathi042 ***@***.***>:

**Reply by vkrathi042** [reply]
dotOS-R-v5.1.1-arm64-gapps-OFFICIAL.img.xz

**Reply by vkrathi042** [reply]
dot os gsi downloaded from their  site
